lms/djangoapps/certificates/views/webview.py

```diff
@@ -490,6 +488,17 @@ def render_cert_by_uuid(request, certificate_uuid):
     except GeneratedCertificate.DoesNotExist:
         raise Http404
 
+def _get_user_national_id(user, preview_mode):
+    try:
+        national_id = NationalId.objects.get(user=user.id)
+        return national_id.get_national_id()
+    except NationalId.DoesNotExist:
+        if preview_mode:
+            national_id = "123456789-AA"
+            return national_id
+        else:
+            national_id= False
```

Comment:
@pepeportela Missing space in `national_id= False`. It should be `national_id = False`